app/src/androidTest/java/io/intrepid/skeleton/testutils/BaseUiTest.java

```diff
@@ -0,0 +1,19 @@
+package io.intrepid.skeleton.testutils;
+
+import org.junit.After;
+import org.junit.Rule;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+import io.intrepid.skeleton.InstrumentationTestApplication;
+
+public class BaseUiTest {
+    @Rule
+    public MockitoRule mockitoRule = MockitoJUnit.rule();
```

Comment:
If we're putting in the capability for instrumentation test mocks, should we give an example like we do on the unit test side?